Blog post: ISOSTS/NISOSTS XML conversion using mnconvert

manuelfuenmayor opened the pull request:
Fixes https://github.com/metanorma/metanorma.org/issues/683

anermina commented:
Thanks for starting this, @manuelfuenmayor ! 

I restructured the text a little bit and added some more tips. Please take a look.

Some notes:
* There are more optional parameters than given in the text. Maybe we should avoid listing them. I propose to just "highlight" the ones we find most useful (I vote for `--output`, `--type`, and `split-bibdata` :) ) and to refer to the README for the complete list.
* I added spaces before STS in "ISOSTS" and "NISOSTS". However, I've seen that the Metanorma website sometimes uses these terms, so if there is some underlying reason for that, please let me know and I'll revert the changes.

manuelfuenmayor commented:
Thanks @anermina for the substantial value you have provided 😃 

> There are more optional parameters than given in the text. Maybe we should avoid listing them. I propose to just "highlight" the ones we find most useful (I vote for --output, --type, and split-bibdata :) ) and to refer to the README for the complete list.

Well, I had the same vision that you present. But  after inspecting the README more deeply, I was reluctant to reference it because I consider that the content is not appealing enough for the average user. In my opinion, it needs more writing work.

And, in regards to the `split-bibdata` option, I have tested it and got unexpected results, that explain here: https://github.com/metanorma/mnconvert/issues/295

> I added spaces before STS in "ISOSTS" and "NISOSTS". However, I've seen that the Metanorma website sometimes uses these terms, so if there is some underlying reason for that, please let me know and I'll revert the changes.

Well, I have seen it in both forms, "ISOSTS" and "ISO STS".  For example, in the [official documentation](https://www.iso.org/schema/isosts/v1.1/doc/index.html) they use "ISOSTS".
But I'd say we leave it as it is right now.

I'm about to upload some minor updates. Please, if you can take a look?
Thanks!

anermina commented:
Looks good to me. Thank you @manuelfuenmayor !
